tests/core/pyspec/eth2spec/test/deneb/block_processing/test_process_execution_payload.py

```diff
@@ -203,3 +203,22 @@ def test_invalid_exceed_max_blobs_per_block(spec, state):
     execution_payload.block_hash = compute_el_block_hash(spec, execution_payload)
 
     yield from run_execution_payload_processing(spec, state, execution_payload, blob_kzg_commitments, valid=False)
+
+
+@with_deneb_and_later
+@spec_state_test
+def test_invalid_parent_block_root(spec, state):
+    execution_payload = build_empty_execution_payload(spec, state)
+
+    opaque_tx, _, blob_kzg_commitments, _ = get_sample_opaque_tx(spec, blob_count=1)
+
+    execution_payload.transactions = [opaque_tx]
+    execution_payload.block_hash = compute_el_block_hash(spec, execution_payload)
+
+    parent_root = spec.get_parent_block_root(state)
+    data = [0xab] * 32
+    incorrect_parent_root = spec.Root.decode_bytes(data)
+    assert incorrect_parent_root != parent_root
```

Comment:
Maybe add assertions in the valid cases too. e.g., in `run_execution_payload_processing` valid cases.

Reply:
this will happen in `spec.process_execution_payload` and the `assert` about the headers matching currently at the end of `run_execution_payload_processing` so I think it is covered in the valid case